control_node/control_node/nodes/control_node.py

```diff
@@ -0,0 +1,334 @@
+"""File: main_node.py
+
+Main ros2 control node for the laser runner removal system. This 
+node uses a state machine to control the general objective of the 
+system. States are things like calibrating the camera laser system,
+finding a specific runner to burn, and burning said runner.  
+"""
+import rclpy
+
+import numpy as np
+
+from simple_node import Node
+import time
+from laser_control.laser_node_client import LaserNodeClient
+from laser_runner_removal.Tracker import Tracker
+from camera_control.camera_node_client import CameraNodeClient
+from yasmin import State, StateMachine, Blackboard
+
+DEBUG = 1
+
+
+class Runner:
+    def __init__(self, pos=None, point=None):
+        self.pos = pos
+        self.point = point
+        self.corrected_laser_point = None
+
+
+class Initialize(State):
+    """Initialization state for setting up resources needed by the system"""
+
+    def __init__(self):
+        State.__init__(self, outcomes=["init_complete"])
+
+    def execute(self, blackboard):
+        blackboard.main_node.logger.info("Entering State Initialize")
```

Comment:
nit: in all of these States we only use the blackboard for passing the main_node around. We can just pass the main_node reference into __init__ instead.

Reply:
I'm going to keep it as is, just because it follows the YASMIN examples and use cases